client/jetpack-connect/remote-credentials.js

```diff
@@ -246,6 +251,12 @@ export class OrgCredentialsForm extends Component {
 							onChange={ this.getChangeHandler( 'password' ) }
 							value={ password || '' }
 						/>
+						{ this.isInvalidPassword() && (
+							<FormInputValidation
+								isError
+								text={ translate( 'Incorrect password. Please try again.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Your password is incorrect, please try again' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👍